README.md

```diff
@@ -89,7 +89,7 @@ Other useful commands:
     # Call ninja manually.
     ./third_party/depot_tools/ninja -C out/debug :all
     # Build a release binary.
-    ./tools/build.py --mode=release :deno
+    DENO_BUILD_MODE=release ./tools/build.py :deno
```

Comment:
Without denying that this patch overall made things a bit more ergonomical, I don't like how verbose this is becoming.

Reply:
Eh, env vars are less code to deal with than argparse. And this command is not often run - I'd prefer to keep it for now.

Reply:
> And this command is not often run

Fair